pkg/controller/atlasproject/cloud_provider_integration.go

```diff
@@ -0,0 +1,385 @@
+package atlasproject
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sort"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+
+	mdbv1 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
```

Comment:
we're not `v1` any more but rather `v2`. `mdbv1` has little meaning. In Atlas CLI we used `akov2` to denote that this is a "Atlas Kubernetes v2.0 top-level package".